languageserver/integration/codelenses.go

```diff
@@ -20,18 +20,37 @@ package integration
 
 import (
 	"fmt"
-
-	"github.com/onflow/flow-go-sdk"
-
 	"github.com/onflow/cadence"
 	jsoncdc "github.com/onflow/cadence/encoding/json"
 	"github.com/onflow/cadence/runtime/ast"
+	"github.com/onflow/cadence/runtime/common"
 	"github.com/onflow/cadence/runtime/sema"
+	"github.com/onflow/flow-go-sdk"
+	"strings"
 
 	"github.com/onflow/cadence/languageserver/conversion"
 	"github.com/onflow/cadence/languageserver/protocol"
 )
 
+const (
+	// Codelens message prefixes
+	prefixOK       = "💡"
+	prefixStarting = "⏲"
+	prefixOffline  = "⚠️"
+	prefixError    = "🚫"
+)
+
+func convertListToCadenceJSON(args []string) string {
+	list := strings.Join(args, ",")
+	cadenceJSON := fmt.Sprintf("[%s]", list)
+	return strings.ReplaceAll(cadenceJSON, "\n", "")
+}
+
+func addressNotExist(address flow.Address) bool {
+	emptyAddress := flow.Address{0}
+	return address.String() == emptyAddress.String()
```

Comment:
flow.Address is an array, you can compare the values directly:
```suggestion
	return address == emptyAddress
```